### ISSUE TITLE
Cannot save "Delineated Watershed" projects

### ISSUE THREAD

**Issue by mmcfarland** [WikiWatershed/model-my-watershed#330]
If you use the Delineate Wathershed feature to generate an Area of Interest, that polygon causes the project not to be saved with:

```json
{"area_of_interest":["Invalid format: string or unicode input unrecognized as WKT EWKT, and HEXEWKB."]}
```

**Comment by caseypt** [comment]
There's a fix in https://github.com/WikiWatershed/model-my-watershed/pull/328

https://github.com/caseypt/model-my-watershed/commit/ef2acb37e030b5f99d1a457bc1d070b41a0d8615

**Comment by mmcfarland** [comment]
Fixed in #328
